lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -333,17 +338,19 @@ def create_service(self, service_name, cluster_name, task_definition, load_balan
             role=role,
             deploymentConfiguration=deployment_configuration,
             placementConstraints=placement_constraints,
-            placementStrategy=placement_strategy)
+            placementStrategy=placement_strategy,
+            healthCheckGracePeriodSeconds=healthcheck_graceperiod)
```

Comment:
This parameter has only existed in botocore since 1.9.12 (March 2018) which is incredibly new. 

This change needs some feature or version detection to ensure this fails with a helpful error message for people who try and use this feature with older versions of botocore